eq-author/.travis.yml

```diff
@@ -8,7 +8,7 @@ services:
 language: node_js
 
 node_js:
-  - "8"
```

Comment:
I guess this file should be removed and the changed pushed up to the root.

Reply:
Yeah this will be the same with each of the subdirectories. I think removing them all together in a separate PR makes sense.

Reply:
Another thing that occurred to me this morning is that we'll have to make changes so that the schema file is imported from the local filesystem as opposed to being an npm dependency. This is because the concourse pipeline has changed to deploy on a new tag in github. So we want to avoid the npm publishing process going forward as this will create a tag and deploy changes to pre-prod.